base/files/memory_mapped_file.cc

```diff
@@ -113,8 +117,11 @@ bool MemoryMappedFile::Initialize(File file,
   if (region != Region::kWholeFile)
     DCHECK_GE(region.offset, 0);
 
-  file_ = std::move(file);
+  #if defined(STARBOARD)
+  file_name_ = file.GetFileName();
```

Comment:
Instead of storing file_name_ as a new field in memory_mapped_file, can we instead use the existing information in file_starboard and get the path of the file from that when we need it in memory_mapped_file_starboard?

Reply:
Well, I'm not sure that's possible. At least, I didn't figure out how. @y4vor Do you know if we can get file path/name from file_starboard?

Reply:
Nvm. I misunderstood the comment earlier.